src/utils/pipelineStatusValues.js

```diff
@@ -0,0 +1,14 @@
+// Pipeline states as defined in
+// https://docs.aws.amazon.com/step-functions/latest/apireference/API_DescribeExecution.html
+// Additional statuses are defined in the API
+
+export default {
+  RUNNING: 'RUNNING',
+  FAILED: 'FAILED',
+  TIMED_OUT: 'TIMED_OUT',
+  ABORTED: 'ABORTED',
+  SUCCEEDED: 'SUCCEEDED',
+
+  // Custom defined statuses defined in the API
+  NOT_CREATED: 'NotCreated',
```

Comment:
do not fix this now because it is too much bother for what it is, but it could have been nicer if this custom state was also in capital letters so that it matches the format of the rest of them.